config-linux.md

```diff
@@ -595,7 +595,7 @@ The values MUST be absolute paths in the [container namespace][container-namespa
     "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
 ```
 
-[container-namespace]: glossary.md#container_namespace
+[container-namespace2]: glossary.md#container_namespace
```

Comment:
The pdf generation tool was generating a warning message so this is one way to avoid that.